src/api-reference/travel/itinerary/trip/trip-resource.markdown

```diff
@@ -652,38 +663,38 @@ This endpoint is used for creating a new trip. To create a new trip, the specifi
 | BookingSource | required	| Y | string	| The name of the booking source for this booking. A booking source is a textual name the system uses to track where a booking took place. This could be a GDS, OTA, Vendor code for a Supplier website, or Supplier Direct Connect API. For TripLink suppliers, this is the supplier's name.
-| StartDateLocal |	optional | Y | dateTime | The start date of the trip in the starting location’s timezone. Format: YYYY-MM-DDThh:mm:ss
-| EndDateLocal | optional | Y | dateTime	| The end date of the trip in the ending location’s timezone. Format: YYYY-MM-DDThh:mm:ss
+| StartDateLocal |	optional | Y | dateTime | The start date of the trip in the starting location’s timezone. Format: YYYY-MM-DDThh:mm:ss.
+| EndDateLocal | optional | Y | dateTime	| The end date of the trip in the ending location’s timezone. Format: YYYY-MM-DDThh:mm:ss.
 | BookedByFirstName	| optional | Y | string	| The first name of the person who booked the trip.
 | BookedByLastName | optional | Y | string	| The last name of the person who booked the trip.
-| Bookings | optional | Y | array	| A parent element that contains a Booking child element for each booking associated with this itinerary.
-| Booking | optional | Y | array	| A child element of the Bookings element which in turn contains the following child elements: Segments, Passengers, RecordLocator, BookingSource, DateModifiedUtc, DateBookedLocal, ItinSourceName, and PassengerCount.
-| Segments | optional	 | Y | array	| List of Segments in this booking. The child elements included in this element vary depending on whether a TMC, third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs and third-party developers**, the Segments element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. For TripLink suppliers, the Segments element contains one or more Air, Car, Hotel, or Ride parent element.
+| Bookings | optional | Y | array	| A parent element that contains a `Booking` child element for each booking associated with this itinerary.
+| Booking | optional | Y | array	| A child element of the `Bookings` element which in turn contains the following child elements: `Segments`, `Passenger`s, `RecordLocator`, `BookingSource`, `DateModifiedUtc`, `DateBookedLocal`, `ItinSourceName`, and `PassengerCount`.
+| Segments | optional	 | Y | array	| List of segments in this booking. The child elements included in this element vary depending on whether a TMC, third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs and third-party developers**, the `Segment`s element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. For TripLink suppliers, the `Segments` element contains one or more Air, Car, Hotel, or Ride parent element.
```

Comment:
Code backticks on first Segments are not quite right. But it seems to render ok?

Reply:
Fixed